google-cloud-spanner/src/main/java/com/google/cloud/spanner/OpenTelemetryApiTracer.java

```diff
@@ -163,8 +165,18 @@ public void attemptCancelled() {
     lastConnectionId = null;
   }
 
+  /**
+   * This method is obsolete. Use {@link #attemptFailedDuration(Throwable, java.time.Duration)}
+   * instead.
+   */
+  @Override
+  @ObsoleteApi("Use attemptFailedDuration(Throwable, java.time.Duration) instead")
+  public void attemptFailed(Throwable error, org.threeten.bp.Duration delay) {
+    attemptFailedDuration(error, toJavaTimeDuration(delay));
+  }
+
   @Override
-  public void attemptFailed(Throwable error, Duration delay) {
+  public void attemptFailedDuration(Throwable error, java.time.Duration delay) {
```

Comment:
nit: Can we import `java.time.Duration`?

Reply:
Done